packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -517,6 +520,16 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         }
     }, [currentVenueName, mapsindoorsSDKAvailable]);
 
+    /*
+     * React on changes to the hideNonMatches prop.
+     */
+    useEffect(() => {
+        if (mapsindoorsSDKAvailable && hideNonMatches) {
+            setHideNonMatches(hideNonMatches);
+        }
+    }, [hideNonMatches, mapsindoorsSDKAvailable]);
```

Comment:
Why is the `mapsindoorsSDKAvailable` dependency needed? 🤔

Reply:
Good catch, it's actually not needed. Removed it.